src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/CaseWorkerServiceImpl.java

```diff
@@ -508,20 +509,29 @@ public CaseWorkerProfile mapCaseWorkerProfileRequest(String idamId,
     // Idam_UP call.
     @SuppressWarnings("unchecked")
     private ResponseEntity<Object> createUserProfileInIdamUP(CaseWorkersProfileCreationRequest cwrdProfileRequest) {
-
+        ResponseEntity<Object> responseEntity;
         Response response = null;
         Object clazz;
         try {
             long time1 = System.currentTimeMillis();
             response = userProfileFeignClient.createUserProfile(createUserProfileRequest(cwrdProfileRequest));
             log.info("{}:: Time taken to call UP is {}", loggingComponentName, (System.currentTimeMillis() - time1));
-            if (response.status() == 201 || response.status() == 409) {
-                clazz = UserProfileCreationResponse.class;
-            } else {
-                clazz = ErrorResponse.class;
-                validationServiceFacade.logFailures(UP_CREATION_FAILED, cwrdProfileRequest.getRowId());
+
+            clazz = (response.status() == 201 || response.status() == 409)
+                    ? UserProfileCreationResponse.class : ErrorResponse.class;
+
+            responseEntity = JsonFeignResponseUtil.toResponseEntity(response, clazz);
+            if (clazz == ErrorResponse.class && nonNull(responseEntity.getBody())) {
```

Comment:
please check if body comes null with status code 4XX, will it log in exception table ?

Reply:
Updated the code to handle this scenario, and it is working as expected.